docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,115 @@
+on most valid blocks it receives, the validator identity account is frequently
+(as often as once per slot, or roughly every 400ms) signing transactions and
+paying fees.  For this reason the validator identity account must be stored as
+a "hot wallet" as a keypair file on the same system the validator process is
+running.
+
+Because a hot wallet is generally less secure than an offline or "cold" wallet,
+we only recommend storing enough SOL on the Identity keypair to cover voting fees
+for some amount of time (perhaps a few weeks or months) and periodically topping
+off your validator identity account from a more secure wallet.
+
+The validator identity is required to be provided when a vote account is created.
+The validator identity can also be changed after an account is created by using
+the `vote-update-validator` command.
+
+### Vote Authority
+
+The _vote authority_ keypair is used to sign each vote transaction the validator
+node wants to submit to the cluster.  This can be the same keypair used as the
+validator identity for the same vote account, or a different keypair.  Because
+the vote authority, like the validator identity, is signing transactions
+frequently, this also must be a hot keypair on the same file system as the
+validator process.
+
+It is recommended to set the vote authority to the same address as the validator
+identity.  If the validator identity is also the vote authority, only one
+signature per vote transaction is needed in order to both sign the vote and pay
+the transaction fee.  Because transaction fees on Solana are assessed
+per-signature, having one signer instead of two will result in half the transaction
+fee paid compared to setting the vote authority and validator identity to two
+different accounts.
+
+The vote authority can be set when the vote account is created.  If it is not
+provided, the vote account with set the Validator Identity as the vote authority
+by default.  The vote authority can be changed later with the
+`vote-authorize-voter` command.
+
+The vote authority can be changed at most once per epoch.  If the authority is
+changed with `vote-authorize-voter`, this will not take effect until the end of
+the current epoch.  To support a smooth transition of the vote signing,
+`solana-validator` allows the `--authorized-voter` argument to be specified
+multiple times.  This allows the validator process to keep voting successfully
+when the network reaches an epoch boundary at with the validator's vote authority
+account changes.
+
+### Withdraw Authority
+
+The _withdraw authority_ keypair is used to withdraw funds from a vote account
+using the `withdraw-from-vote-account` command.  Any network rewards a validator
+earns are deposited into the vote account and are only retrievable by signing
+with the withdraw authority keypair.
+
+The withdraw authority is also required to sign any transaction to change
+a vote account's [commission](#commission).
```

Comment:
I believe it is required to sign for _all_ account settings modifying transactions

Reply:
It is needed to change the withdrawer (clearly) and also to change the validator identity, but not needed to set a new vote authority.